golang/x/relay/client/cli/tx.go

```diff
@@ -66,6 +66,101 @@ func GetCmdIngestHeaderChain(cdc *codec.Codec) *cobra.Command {
 	}
 }
 
+// GetCmdIngestDifficultyChange creates a CLI command to ingest a difficulty change
+func GetCmdIngestDifficultyChange(cdc *codec.Codec) *cobra.Command {
+	return &cobra.Command{
+		Use:   "ingestdiffchange <prev epoch start> <json list of headers>",
+		Short: "ingest a difficulty change",
+		Long:  "Ingest a difficulty change",
+		Args:  cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cliCtx := context.NewCLIContext().WithCodec(cdc)
+
+			txBldr := auth.NewTxBuilderFromCLI().WithTxEncoder(utils.GetTxEncoder(cdc))
+
+			var prevEpochStart types.Hash256Digest
+			jsonErr := json.Unmarshal([]byte(args[1]), &prevEpochStart)
+			if jsonErr != nil {
+				return jsonErr
+			}
+
+			var headers []types.BitcoinHeader
+			jsonErr = json.Unmarshal([]byte(args[1]), &headers)
+			if jsonErr != nil {
+				return jsonErr
+			}
+
+			msg := types.NewMsgIngestDifficultyChange(
+				cliCtx.GetFromAddress(),
+				prevEpochStart,
+				headers,
+			)
+			err := msg.ValidateBasic()
+			if err != nil {
+				return err
+			}
+
+			return utils.GenerateOrBroadcastMsgs(cliCtx, txBldr, []sdk.Msg{msg})
+
+		},
+	}
+}
+
+// GetCmdMarkNewHeaviest creates a CLI command to update best known digest and LCA
+func GetCmdMarkNewHeaviest(cdc *codec.Codec) *cobra.Command {
+	return &cobra.Command{
+		Use:   "marknewheaviest <ancestor> <currentBest> <newBest> [limit]",
+		Short: "Updates best known digest and LCA",
+		Long:  "Updates best known digest and LCA",
```

Comment:
Add note that ancestor, current best, and new best are hex, and limit is an integer